addons/binding/org.openhab.binding.avmfritz/ESH-INF/i18n/avmfritz_de.properties

```diff
@@ -0,0 +1,149 @@
+
+thing-type.avmfritz.FRITZ_DECT_210.description = FRITZ!DECT 210 schaltbare Steckdose. Dient zur Steuerung der integrierten Steckdose und liefert Daten wie z.B. Temperatur und Energie.
+
+thing-type.avmfritz.FRITZ_DECT_300.description = FRITZ!DECT 300 Heizk�rperregler. Dient zur Steuerung von Heizk�rpern und liefert Daten wie z.B. Temperatur.
+
+thing-type.avmfritz.Comet_DECT.description = Comet DECT Heizk�rperregler. Dient zur Steuerung von Heizk�rpern und liefert Daten wie z.B. Temperatur.
+
+thing-type.avmfritz.FRITZ_Powerline_546E.description = FRITZ!Powerline 546E schaltbare Steckdose. Dient zur Steuerung der integrierten Steckdose und liefert Daten wie z.B. Temperatur.
+
+# thing types config
+thing-type.config.avmfritz.fritzdevice.ain.description = Die AHA ID (AIN) zur Identifikation des FRITZ! Ger�tes.
+
+# channel types
+channel-type.avmfritz.mode.label = Modus des Ger�tes
+channel-type.avmfritz.mode.description = Zeigt den aktuellen Modus des Ger�tes an (MANUAL/AUTOMATIC).
+channel-type.avmfritz.mode.state.option.MANUAL = Manuell
+channel-type.avmfritz.mode.state.option.AUTOMATIC = Automatisch
+
+channel-type.avmfritz.locked.label = Schalten des Ger�tes (extern)
+channel-type.avmfritz.locked.description = Zeigt an, ob die Sperre f�r Schalten des Ger�tes �ber extern aktiviert ist.
+
+channel-type.avmfritz.temperature.label = Temperatur
+channel-type.avmfritz.temperature.description = Zeigt die aktuelle Temperatur (in �C) an.
+
+channel-type.avmfritz.energy.label = Gesamtverbrauch
+channel-type.avmfritz.energy.description = Zeigt den akkumulierten Gesamtverbrauch (in kWh) an.
+
+channel-type.avmfritz.power.label = Leistung
+channel-type.avmfritz.power.description = Zeigt die aktuelle Leistung (in W) an.
+
+channel-type.avmfritz.outlet.label = Steckdose
+channel-type.avmfritz.outlet.description = Erm�glicht die Steuerung der Steckdose (ON/OFF).
+
+channel-type.avmfritz.actual_temp.label = Temperatur
+channel-type.avmfritz.actual_temp.description = Zeigt die aktuell gemessene Temperatur (in �C) des Heizk�rperreglers an.
+
+channel-type.avmfritz.set_temp.label = Solltemperatur
+channel-type.avmfritz.set_temp.description = Erm�glicht die Steuerung der Solltemperatur (in �C) des Heizk�rperreglers.
+
+channel-type.avmfritz.eco_temp.label = Absenktemperatur
+channel-type.avmfritz.eco_temp.description = Zeigt die aktuell eingestellte Absenktemperatur (in �C) des Heizk�rperreglers an.
+
+channel-type.avmfritz.comfort_temp.label = Komforttemperatur
+channel-type.avmfritz.comfort_temp.description = Zeigt die aktuell eingestellte Komforttemperatur (in �C) des Heizk�rperreglers an.
+
+channel-type.avmfritz.radiator_mode.label = Modus des Heizk�rperreglers
+channel-type.avmfritz.radiator_mode.description = Erm�glicht die Steuerung des aktuellen Modus des Heizk�rperreglers (ON/OFF/COMFORT/ECO/BOOST).
```

Comment:
I added those values here because if you want to define a `Selection` item in a `Sitemap` you need those possible values. I am not sure if you are allowed to send the translated values to the channel via `Selection` item or in a rule using `sendCommand` or `postUpdate`.

´´´csv
Selection item=fritzBedroomRadiatorMode mappings=["ON"="An", "OFF"="Aus", "COMFORT"="Komfort", "ECO"="Eco", "BOOST"="Boost"]
´´´